addons/medical_treatment/CfgVehicles.hpp

```diff
@@ -38,6 +38,25 @@ class CfgVehicles {
             };
         };
     };
+    class ACE_bodyBagObject_new: ACE_bodyBagObject {
+        hiddenSelections[] = {"camo"};
+        model = "\A3\Props_F_Orange\Humanitarian\Camps\Bodybag_01_F.p3d";
+    };
+
+    class ACE_bodyBag_black: ACE_bodyBagObject_new {
```

Comment:
I think we should just move this back to `ACE_bodyBagObject`?

just in-case people have mods/scripts that use that specific classname
like "ACE_bodyBagObject" in area detection or cargo loaded
maybe keep the old model as a new classname `ACE_bodyBagObject_original`